src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -344,6 +344,22 @@ def test_custom_image_with_plan(self, resource_group):
         self.cmd('vm show -g {rg} -n vm1',
                  checks=self.check('plan.name', '{plan}'))
 
+    @ResourceGroupPreparer()
+    def test_vm_create_with_market_place_image(self, resource_group, resource_group_location):
+        self.kwargs.update({
+            'location': resource_group_location,
+            'publisher': 'kemptech',
+            'offer': 'vlm-azure',
+            'sku': 'basic-byol',
+            'plan': 'basic-byol'
+        })
+        urn = '{}:{}:{}:7.2.362142710'.format(self.kwargs['publisher'], self.kwargs['offer'], self.kwargs['sku'])
+        self.cmd('vm image show --urn ' + urn, checks=self.check('plan.name', self.kwargs['plan']))
+        self.cmd('vm image accept-term -p {publisher} --offer {offer} --plan {plan}', checks=self.check('accepted', True))
+        # repeat the same command using --urn
+        self.cmd('vm image accept-term --urn ' + urn, checks=self.check('accepted', True))
+        self.cmd('vm create -g {rg} -n vm1 --no-wait --image ' + urn)
```

Comment:
Same.  Recommend putting urn into kwargs and using the normal syntax to make it a little cleaner.